### DISCUSSION TITLE
[Step-by-step] Guide On Perfect Bed Adhesion and Elephant Foot Removal

### DISCUSSION THREAD

**Discussion by sn4k3** [sn4k3/UVtools#534] [Tutorials]
This guide was copied from [honzamrazek blog](https://blog.honzamrazek.cz/2022/07/step-by-step-guide-on-perfect-bed-adhesion-and-elephant-foot-removal-in-uvtools-3) 
Also [read the cause here](https://blog.honzamrazek.cz/2022/01/prints-not-sticking-to-the-build-plate-layer-separation-rough-surface-on-a-resin-printer-resin-viscosity-the-common-denominator)

**Pre-notes:** 
- For CTB files is important that you have version 4 of the file format, if you slice on Chitubox it's v4 by default. On lychee you need to ensure you select v4.
- It won't work for Anycubic nor Creality Halot printers since there are only 1 global wait time property to set all layers

![image](https://user-images.githubusercontent.com/113281/183948328-e8207f7f-a84f-413b-9f87-f2c78389aaf9.png)
![image](https://user-images.githubusercontent.com/113281/183948425-6b0bb7d1-169f-4a93-b9ff-91931be41491.png)
![image](https://user-images.githubusercontent.com/113281/183948547-38cff178-633f-4567-be2a-7ba15e1d44fb.png)
![image](https://user-images.githubusercontent.com/113281/183948647-09587566-15f1-43d6-8bdc-ac3586f355a4.png)
![image](https://user-images.githubusercontent.com/113281/183948710-a4006a52-9ba4-4d22-8b36-49a76e9e221e.png)
![image](https://user-images.githubusercontent.com/113281/183948777-d9989112-bab0-43cd-8214-3d6b72e6c12a.png)
![image](https://user-images.githubusercontent.com/113281/183948823-b835ae06-38ff-4ecc-a8e1-573fc119aa1c.png)
![image](https://user-images.githubusercontent.com/113281/183948880-b386af04-4271-4774-890b-1d263980668f.png)
![image](https://user-images.githubusercontent.com/113281/183948934-7902c222-8a3f-4424-8c18-59991254fb6f.png)
![image](https://user-images.githubusercontent.com/113281/183948984-d5426536-ed5c-43c0-bf57-645822670062.png)
![image](https://user-images.githubusercontent.com/113281/183949026-f4acfb2b-9c20-4985-8300-af3057906c38.png)
![image](https://user-images.githubusercontent.com/113281/183949075-d36ecabe-1ac4-4588-9fe6-d6eb9ab59f61.png)

Last, save and print :)

**Comment by michox** [comment]
@sn4k3 awesome feature, I will be using this a lot since I have a functional print with a flat plate and then a 90 degree rib that needs to be a precise flat surface and I keep getting blooming on the transition between the plate and the rib. I would like to understand the proportional features a little better, e.g. how exactly does the proportional area feature work? The tool tips are not quite enough for me to understand and use the features properly, it would be awesome if you could provide some insights on this!

**Reply by sn4k3** [reply]
The proportial method is a simple math "the rule of three".
Based on current layer pixels it set the wait time.

Example: You set to 1000px2 to correspond to 3s wait time.
If current layer is 1000px2 the wait time will be 3s. But if area is smaller or larger it will adjust time based on area such as: 

Wait time = Current layer area * your set wait time / Your set layer area

eg: 500px2 * 3s / 1000px2 = 1,5 s wait time on a layer with half the mass. So it adjust time based on mass.
